presto-product-tests/src/main/java/com/facebook/presto/tests/cli/PrestoCliTests.java

```diff
@@ -144,7 +144,8 @@ public void shouldRunBatchQuery()
         assertThat(trimLines(presto.readRemainingOutputLines())).containsAll(nationTableBatchLines);
     }
 
-    @Test(groups = CLI, timeOut = TIMEOUT)
+    //@Test(groups = CLI, timeOut = TIMEOUT)
+    // TODO: Un-quarantine after Jetty update: https://github.com/eclipse/jetty.project/pull/937
```

Comment:
It must be fixed with the Jetty update. My refactor to the CLI tests just revealed the issue, because similary to the changes @brian-rickman have made, it enables actual kerberos testing.